languages/elixir/reference/exercise-concepts/secret-handshake.md

```diff
@@ -0,0 +1,53 @@
+# Concepts for secret-handshake
+
+## Optimal solution
+```elixir
+defmodule SecretHandshake do
+  use Bitwise
+
+  @doc """
```

Comment:
As Elixir comments are special, they definitely warrant including a "Comments" Concept.

Reply:
Nice catch, we are missing that as a pointed concept